app/code/local/Magmodules/Webwinkelconnect/Model/System/Config/Source/Language.php

```diff
@@ -30,11 +30,11 @@ public function toOptionArray()
         $lan = array();
         $lan[] = array('value' => '', 'label' => Mage::helper('webwinkelconnect')->__('Use default'));
         $lan[] = array('value' => 'cus', 'label' => Mage::helper('webwinkelconnect')->__('Based on customer country'));
-        $lan[] = array('value' => 'nld', 'label' => Mage::helper('webwinkelconnect')->__('Dutch'));
-        $lan[] = array('value' => 'eng', 'label' => Mage::helper('webwinkelconnect')->__('English'));
-        $lan[] = array('value' => 'deu', 'label' => Mage::helper('webwinkelconnect')->__('German'));
-        $lan[] = array('value' => 'fra', 'label' => Mage::helper('webwinkelconnect')->__('French'));
-        $lan[] = array('value' => 'spa', 'label' => Mage::helper('webwinkelconnect')->__('Spanish'));
+        $lan[] = array('value' => 'NL', 'label' => Mage::helper('webwinkelconnect')->__('Dutch'));
+        $lan[] = array('value' => 'EN', 'label' => Mage::helper('webwinkelconnect')->__('English'));
+        $lan[] = array('value' => 'DE', 'label' => Mage::helper('webwinkelconnect')->__('German'));
+        $lan[] = array('value' => 'FR', 'label' => Mage::helper('webwinkelconnect')->__('French'));
+        $lan[] = array('value' => 'ES', 'label' => Mage::helper('webwinkelconnect')->__('Spanish'));
```

Comment:
What is the reasoning for capitalizing the `value` here?
What about Bulgarian, don't you like it?